src/Uno.Toolkit.UI/Controls/NavigationBar/AppBarButtonRenderer.Android.cs

```diff
@@ -94,8 +94,6 @@ protected override IEnumerable<IDisposable> Initialize()
 				yield return element.RegisterDisposableNestedPropertyChangedCallback(
 					(s, e) => Invalidate(),
 					new[] { AppBarButton.LabelProperty },
-					new[] { AppBarButton.IconProperty },
-					new[] { AppBarButton.IconProperty, BitmapIcon.UriSourceProperty },
```

Comment:
The PR keeps the intent of the original code, but I've looked through the Renderer's code and I don't see a reason why it needs to subscribe to Icon.UriSource. It doesn't use the UriSource at all, which makes sense since it's a part of the BitmapIcon itself, and has nothing to do with the owning AppBarButton. I tested removing this subscription and changing the UriSource after loading and nothing seems to break.